src/blocks/scratch3_motion.js

```diff
@@ -258,12 +258,16 @@ class Scratch3MotionBlocks {
         util.target.setXY(util.target.x, y);
     }
 
+    roundCoord (value, places) {
+        return Math.round(value * Math.pow(10, places)) / Math.pow(10, places);
```

Comment:
Using a constant variable in place of the Math.pow calls I think would perform better and let the constant names better describe why the rounding is needed? Does that seem fair?